TreeTracker/UI/Views/Authentication/SignIn/SignInViewController.swift

```diff
@@ -23,24 +24,37 @@ class SignInViewController: UIViewController, KeyboardDismissing, AlertPresentin
             usernameTextField.returnKeyType = .done
             usernameTextField.placeholder = L10n.SignIn.TextInput.PhoneNumber.placeholder
             usernameTextField.validationState = .normal
+            usernameTextField.backgroundColor = .black
+            usernameTextField.layer.borderColor = UIColor.lightGray.cgColor
+            usernameTextField.layer.cornerRadius = 10.0
+            usernameTextField.layer.borderWidth = 1.0
+            usernameTextField.clipsToBounds = true
         }
     }
     @IBOutlet private var usernameSegmentedControl: UISegmentedControl! {
         didSet {
             usernameSegmentedControl.heightAnchor.constraint(equalToConstant: 50).isActive = true
-            usernameSegmentedControl.setImage(Asset.Assets.phone.image, forSegmentAt: 0)
-            usernameSegmentedControl.setImage(Asset.Assets.mail.image, forSegmentAt: 1)
-            usernameSegmentedControl.setTitleTextAttributes([.foregroundColor: Asset.Colors.grayDark.color], for: .selected)
-            usernameSegmentedControl.setTitleTextAttributes([.foregroundColor: Asset.Colors.grayLight.color], for: .normal)
+            let attr = NSDictionary(object: UIFont(name: "Montserrat-Bold", size: 16)!, forKey: NSAttributedString.Key.font as NSCopying)
+            usernameSegmentedControl.setTitleTextAttributes(attr as? [ NSAttributedString.Key : AnyObject ], for: .normal)
+            if #available(iOS 13.0, *) {
```

Comment:
How deos this look in iOS11/12

Reply:
Hey Alex, are you referring to this ?
if #available(iOS 13.0, *) {
                usernameSegmentedControl.selectedSegmentTintColor = Asset.Colors.primaryGreen.color
            }

Reply:
@lexypaul13 are you able to install older simulators and run on those?

Reply:
Yes, here's a screen shot of it on the iPhone 8
![Simulator Screen Shot - iPhone 8 - 2021-11-08 at 15 13 43](https://user-images.githubusercontent.com/55071531/140811359-5c7a96c7-836c-47a1-a436-c3c4f5a13b87.png)
.

Reply:
@AlexCornforth Is this color okay, and is it appropriate to change the color by the key path
<img width="1160" alt="Screen Shot 2021-11-15 at 10 27 30 AM" src="https://user-images.githubusercontent.com/55071531/141808868-fd96e3ad-7f42-40e2-bbac-9e4dcabdad79.png">
?

Reply:
I think it would be better to set the placeholder colour in code - i generally just use storyboards for layout and nothing else. More storyboard changes = increased risk of nasty conflicts. Also code is much easier to review than the giant storyboard xml files.

Reply:
Hey Alex, I've changed the placeholder text color using the code below.
UsernameTextField.attributedPlaceholder = NSAttributedString( string: L10n.SignIn.TextInput.PhoneNumber.placeholder,attributes: [NSAttributedString.Key.foregroundColor: Asset.Colors.grayLight.color]
           
![Simulator Screen Shot - iPhone 13 - 2021-11-29 at 14 27 17](https://user-images.githubusercontent.com/55071531/143930188-c90c5210-cdbf-4938-962c-704de538b448.png)
 )